tekton/overlays/nightly-releases/fetch-nightly-components.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fetch-component-nightly-releases
+spec:
+  workspaces:
+  - name: source
+    mountPath: /go/src/github.com/tektoncd/operator
+  steps:
+  - name: fetch-component-nightlies
+    image: docker.io/curlimages/curl:7.75.0
+    securityContext:
+      runAsUser: 0
+    script: |
+      #!/usr/bin/env sh
+      set -ex
+      cd /go/src/github.com/tektoncd/operator
+      components="pipeline trigger dashboard"
+      for component in ${components}; do
+        componentPath=cmd/kubernetes/operator/kodata/tekton-${component}
+        rm -rf ${componentPath}/*
+        mkdir -p ${componentPath}/nightly
+        if [[ ${component} == "dashboard" ]]; then
+          curl -L -o ${componentPath}/nightly/release.notags.yaml https://storage.googleapis.com/tekton-releases-nightly/${component}/latest/tekton-dashboard-release.yaml
+          wc -l ${componentPath}/nightly/release.notags.yaml
+          sed -i '/aggregationRule/,+3d' ${componentPath}/nightly/release.notags.yaml
+          wc -l ${componentPath}/nightly/release.notags.yaml
+          continue
+        fi
+        curl -L -o ${componentPath}/nightly/release.notags.yaml https://storage.googleapis.com/tekton-releases-nightly/${component}/latest/release.notags.yaml
```

Comment:
I wonder if we could have a param/flag in this task about using `notags` or not? 
Any reason for fetching only the notags version of the releases?

Reply:
ooops 😛 
fixed 👍

Reply:
few months back notags seem to work on all kubernetes platforms.
i shall discuss this in today's operator workgroup meeting and make call.

if we decide to add a param, then i shall make another pr. 🧑‍💻

Reply:
@afrittoli the only reason for using `notags` is to be sure it works with all kubernetes CRI implementation 😝.